examples/meta/src/evaluation/cross_validation_linear_svm.sg

```diff
@@ -0,0 +1,38 @@
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
```

Comment:
I would definitely also evaluate the test error, then you can say in the notebook that xvalidation can be used to estimate the test error. THis requires loading test data

Reply:
Sure, I've now included into the example evaluating the test error.